wasmcloud-chat/actors/messages/Makefile

```diff
@@ -0,0 +1,34 @@
+COLOR ?= always # Valid COLOR options: {always, auto, never}
+CARGO = cargo --color $(COLOR)
+TARGET = target/wasm32-unknown-unknown
+DEBUG = $(TARGET)/debug
+RELEASE = $(TARGET)/release
+KEYDIR ?= .keys
+VERSION = $(shell cargo metadata --no-deps --format-version 1 | jq -r '.packages[].version')
+
+.PHONY: all build check clean doc test update
+
+all: build
+
+build:
+	@$(CARGO) build	
+	wash claims sign $(DEBUG)/messages.wasm -c wasmcloud:httpserver --name "messages" --ver $(VERSION) --rev 0
```

Comment:
Messages will need `wasmcloud:messaging` rather than httpserver right? (based off the flow diagram)

Reply:
this is all still scaffolding. I haven't modified any of the Makefiles to correspond to the actual responsibilities of the actors yet.